docs/src/components/Note.astro

```diff
@@ -34,15 +34,12 @@ const { type = 'tip', title } = Astro.props;
   } */
 
   .note.type-tip {
-    --color: var(--color-green);
-    --color-rgb: var(--color-green-rgb);
+    --color: hsla(var(--color-green), 1);
   }
   .note.type-warning {
-    --color: var(--color-yellow);
-    --color-rgb: var(--color-yellow-rgb);
+    --color: hsla(var(--color-yellow), 1);
   }
   .note.type-error {
-    --color: var(--color-red);
-    --color-rgb: var(--color-red-rgb);
+    --color: hsla(var(--color-red), 1);
   }
```

Comment:
I know these aren't used at the moment but you could do `--color: var(--color-green);` and then the component can set the opacity as needed!

Reply:
I agree, I'll change it.